plugins/snapshots/devmapper/blkdiscard/blkdiscard.go

```diff
@@ -25,6 +25,15 @@ func Version() (string, error) {
 	return blkdiscard("--version")
 }
 
+// Search blkdiscard binary.
+func CheckBlkdiscardBinary() error {
```

Comment:
minor, but can we change this function name? As it is now, when it is used it repeats itself (blkdiscard.CheckBlkdiscardBinary()); I suggest renaming to simply `CheckBinary()`.

Reply:
done.thanks